packages/std/src/traits.rs

```diff
@@ -65,7 +70,7 @@ pub trait Storage: ReadonlyStorage {
 ///
 /// We can use feature flags to opt-in to non-essential methods
 /// for backwards compatibility in systems that don't have them all.
-pub trait Api: Copy + Clone + Send {
+pub trait Api {
```

Comment:
Nice. I was thinking about this change too some time ago. This keeps us more flexible in the future.

In the VM, the API needs to be copyable to allow using it easily in import implementations.

Reply:
We can look at the VM changes later. Maybe with similar changes to `Backend` in the VM, we only need `&Api` and it doesn't matter if `Api` is copyable or not. But that is a different question